packages/shared/components/MenuLogin/types.ts

```diff
@@ -26,6 +26,9 @@ export type MenuLoginProps = {
   transformOrigin?: any;
   placeholder?: string;
   required?: boolean;
+  disabled?: boolean;
+  buttonText?: string;
+  info?: string;
```

Comment:
> The SET DATABASE NAME button looks a bit different than in the issue, we can create a specialised version of MenuLoginThemeProvider if it's needed.

Yeah, I totally forgot about that side effect of reusing MenuLogin. I have two ideas which will let us customize MenuLogin without having to use `MenuLoginThemeProvider`:

1. Add `caret` boolean prop to `MenuLogin` that will not render the caret if false, defaults to true.
   * I think that caret makes sense only when submitting the input will transition you somewhere + when we provide some suggestions (like on our DocumentCluster or in the server list in the Web UI). IMHO it feels weird for the db name and port buttons, especially since we won't even provide suggestions at the beginning.
   * idk why the icon is called `CarrotDown`. 😅 Even ["caret"](https://en.wikipedia.org/wiki/Caret_(computing)) is not the right name for it but I think "caret down" is a common name for it in icon sets.
3. Instead of using `ButtonBorder` in `MenuLogin`, use `Button` and make `buttonKind` a prop with a default of `"border"`.

I think those will elegantly solve this problem, won't they?

Reply:
No longer needed.